apps/roam/src/components/settings/NodeConfig.tsx

```diff
@@ -40,6 +120,68 @@ const NodeConfig = ({
   });
 
   const [selectedTabId, setSelectedTabId] = useState<TabId>("general");
+  const [tagError, setTagError] = useState("");
+  const [formatError, setFormatError] = useState("");
+  const isValid = !tagError && !formatError;
+
+  const {
+    value: tagValue,
+    handleChange: handleTagChange,
+    handleBlur: handleTagBlurFromHook,
+  } = useDebouncedRoamUpdater(tagUid, node.tag, isValid);
+  const {
+    value: formatValue,
+    handleChange: handleFormatChange,
+    handleBlur: handleFormatBlurFromHook,
+  } = useDebouncedRoamUpdater(formatUid, node.format, isValid);
+
+  const getCleanTagText = (tag: string): string => {
+    return tag.replace(/^#+/, "").trim().toUpperCase();
+  };
+
+  const validate = useCallback((tag: string, format: string) => {
+    const cleanTag = getCleanTagText(tag);
+
+    if (!cleanTag) {
+      setTagError("");
+      setFormatError("");
+      return;
+    }
+
+    const formatWithoutPlaceholders = format.replace(/{[^}]+}/g, "");
+    const formatUpper = formatWithoutPlaceholders.toUpperCase();
+    const formatParts = formatUpper.split(/[^A-Z0-9]/);
+    const hasConflict = formatParts.includes(cleanTag);
+
+    if (hasConflict) {
+      const formatForMessage = formatWithoutPlaceholders
+        .trim()
+        .replace(/(\s*-)*$/, "");
+      setFormatError(
+        `Format "${formatForMessage}" conflicts with tag: "${tag}". Please use some other format.`,
```

Comment:
Let's be slightly more specific.  Something like "`${tag}` cannot also be in Format"